deployment/helm/ditto/README.md

```diff
@@ -1,49 +1,137 @@
 ```
 
-## Installation
-To install the Ditto chart with the name `"my-ditto"`, perform:
+Alternatively, a YAML file that specifies the values for the parameters can be provided while installing the chart.
+
+## Chart configuration options
+
+Please consult the [values.yaml](https://github.com/eclipse-ditto/ditto/blob/master/deployment/helm/ditto/values.yaml) 
+for all available configuration options of the Ditto Helm chart.  
+
+### Scaling options
 
-```shell
-helm install -f values.yaml -f local-values.yaml my-ditto . --wait
-kubectl port-forward svc/my-ditto-nginx 8080:8080
+Please note the defaults the chart comes with:
+* the default deploy 1 instance per Ditto service
+* each Ditto service is configured to require:
+  * 0.5 CPUs
+  * 1024 MiB of memory
+
+Adjust this to your requirements, e.g. scale horizontally by configuring a higher `replicaCount` or vertically by 
+configuring more resources.
+
+## Advanced configuration options
+
+Even more configuration options, not exposed to the `values.yaml`, can be configured using either environment variables
+or Java "System properties".  
+To inspect all available configuration options, please inspect Ditto's service configurations:
+
+* [policies.conf](https://github.com/eclipse-ditto/ditto/blob/master/policies/service/src/main/resources/policies.conf)
+* [things.conf](https://github.com/eclipse-ditto/ditto/blob/master/things/service/src/main/resources/things.conf)
+* [things-search.conf](https://github.com/eclipse-ditto/ditto/blob/master/thingsearch/service/src/main/resources/search.conf)
+* [connectivity.conf](https://github.com/eclipse-ditto/ditto/blob/master/connectivity/service/src/main/resources/connectivity.conf)
+* [gateway.conf](https://github.com/eclipse-ditto/ditto/blob/master/gateway/service/src/main/resources/gateway.conf)
+
+
+### Configuration via environment variables
+
+In order to provide an environment variable config overwrite, simply put the environment variable in the `extraEnv` 
+of the Ditto service you want to specify the configuration for.
+
+E.g. if you want to configure the `LOG_INCOMING_MESSAGES` for the `things` service to be disabled:  
+```yaml
+things:
+  # ...
+  extraEnv:
+    - name: LOG_INCOMING_MESSAGES
+      value: "false"
 ```
 
-Now, you can access Ditto on [http://localhost:8080](http://localhost:8080) - have fun.
+### Configuration via system properties
+
+Not all Ditto/Akka configuration options have an environment variable overwrite defined in the configuration.  
+For configurations without such an environment variable overwrite, the option can be configured via Java system property.  
+The documentation on how this works can be found in the 
+[HOCON documentation](https://github.com/lightbend/config/blob/main/HOCON.md#conventional-override-by-system-properties),
+which is the configuration format Ditto uses.
+
+E.g. if you want to would like to adjust the `journal-collection` name which the `things` service uses to write its
```

Comment:
E.g. if you **want to ~~would like~~** to adjust the `journal-collection` name which the `things` service uses to write its